robot/Cumulus/tests/browser/zpayment_allocations/enable_pmt_alloc_and_test.robot

```diff
@@ -64,55 +90,50 @@ Update GAU Allocations and Verify Payment Allocations Sync
 Enable Payment Allocations
     &{def_gau} =  API Create GAU    Name=default gau
     Set Global Variable     &{def_gau}    &{def_gau}
```

Comment:
Why is &{def_gau} called twice while setting the global variables? Is there any reason behind it?

Reply:
as specified on the call used it bcoz of syntax but converted to Suite Variable as not needed as global

Reply:
We went over this in call.